### PR TITLE
Skaffold will by default deploy to local context

### PR DESCRIPTION
**Pull request by tnishida1**
- Adds deploy section by default so skaffold will always build
- changing the folder from chart to charts to conform with helm

### CODE REVIEW

File: skaffold.yaml [hunk 1]
```diff
@@ -8,15 +8,25 @@ build:
...
-  - name: jenkins
+  - name: build
     build:
       artifacts:
         - image: knowledge-share-app
           context: .
           docker: {}
       tagPolicy:
         envTemplate:
-          template: "{{ .DOCKER_REGISTRY }}/liatrio/{{ .IMAGE_NAME }}:{{ .GIT_COMMIT }}"
```
Comment (cplee) [inline skaffold.yaml:20]: Is this change right?

File: Jenkinsfile [hunk 2]
```diff
@@ -20,7 +20,7 @@ def skaffoldBuild() {
             sh "echo 'sonar.login=${sonarqubeToken}' >> sonar.properties"
         }
         docker.withRegistry("https://${DOCKER_REGISTRY}", 'artifactory-credentials') {
-            sh "skaffold build -p jenkins"
+            sh "skaffold build -p build"
```
Comment (cplee) [inline Jenkinsfile]: Why this change?

File: skaffold.yaml [hunk 3]
```diff
@@ -8,15 +8,25 @@ build:
...
     gitCommit: {}
   local: {}
+deploy:
+  helm:
+    releases:
+    - name: knowledge-share-app
+      chartPath: charts/knowledge-share-app
+      setValueTemplates:
+        image.repository: "knowledge-share-app"
+        image.tag: "latest"
```
Comment (cplee) [inline skaffold.yaml]: Change latest to use commit id

**Review by cplee — CHANGES_REQUESTED** [review]

File: skaffold.yaml [hunk 4]
```diff
@@ -8,15 +8,25 @@ build:
...
+  - name: build
     build:
       artifacts:
         - image: knowledge-share-app
           context: .
           docker: {}
       tagPolicy:
         envTemplate:
-          template: "{{ .DOCKER_REGISTRY }}/liatrio/{{ .IMAGE_NAME }}:{{ .GIT_COMMIT }}"
+            template: "{{ .DOCKER_REGISTRY }}/artifactory/{{ .IMAGE_NAME }}:{{ .GIT_COMMIT }}"
```
Comment (ChrisSchreiber) [inline skaffold.yaml]: Is this correct? Don't we want our images to be something like liatrio/knowledge-share-app not artifactory/knowledge-share-app?

File: Jenkinsfile (hunk 2 above)
Comment (tnishida1) [reply Jenkinsfile]: I was following the sprint 8 narrative
"Skaffold configuration will include a profile named ‘build’ that will disable deployment and will enable push to artifactory"

Comment (cplee) [reply Jenkinsfile]: My bad.   We changed the name this morning.

Comment (cplee) [reply Jenkinsfile]: My bad.   We changed the name this morning.

Comment (tnishida1) [reply Jenkinsfile]: all good 👍 should I change the profile back to jenkins

File: skaffold.yaml (hunk 4 above)
Comment (tnishida1) [reply skaffold.yaml]: you're correct, I was confusing the docker_registry and the url point to push the images to. Changing it back to use 'liatrio'

File: skaffold.yaml (hunk 3 above)
Comment (tnishida1) [reply skaffold.yaml]: changed to use GIT_COMMIT env

**Review by cplee — APPROVED** [review]
